webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -30,11 +31,8 @@ type Props = {
     featureName: string;
     minimumSKURequiredForFeature: LicenseSkus;
 
-    titleID: string;
-    titleDefault: string;
-
-    copyID: string;
-    copyDefault: string;
+    title: MessageDescriptor;
```

Comment:
A lot of these changes are replacing ID/default props with a single prop that takes a message descriptor